0.8.3/docker-entrypoint.sh

```diff
@@ -65,10 +65,8 @@ fi
 
 # If we are running Vault, make sure it executes as the proper user.
 if [ "$1" = 'vault' ]; then
-    # If the config dir is bind mounted then chown it
-    if [ "$(stat -c %u /vault/config)" != "$(id -u vault)" ]; then
-        chown -R vault:vault /vault/config || echo "Could not chown /vault/config (may not have appropriate permissions)"
-    fi
+    # If the config dir is bind mounted confirm it's contents are readable
+    su -m vault -c 'find /vault/config >/dev/null 2>&1 || echo "There are files in /vault/config that are not readable by user vault"'
```

Comment:
This won't actually do what you expect -- find won't return a non-zero code simply because a file is not readable (it will for directories only since it can't recurse).